internal/pkg/plan/push.go

```diff
@@ -20,6 +20,10 @@ func Push(diffResults *diff.Results, changeDescription string) (*DiffPlan, error
 		case diff.ResultEqual:
 			// nop
 		case diff.ResultNotEqual:
+			// SKIP: if only Relations have changed + no changed relations on the API side
+			if result.ChangedFields.String() == "relations" && !result.ChangedFields.Get("relations").HasPath("InApi") {
```

Comment:
To iste `push`, ak je zmena iba v `Relations`, ale nie je ziadna zmna v `InApi` relations, tak sa operacia `ActionSaveRemote` nevytvori.